zinc/src/sbt-test/source-dependencies/trait-trait-211/mirtest/incOptions.properties

```diff
@@ -0,0 +1,2 @@
+apiDebug = true
```

Comment:
Should be removed after fix passes CI.

Reply:
Actually I think it's ok for this stay, since the debug logs are displayed only when the test fails.